docs/index.rst

```diff
@@ -14,11 +18,32 @@ Welcome to Earthpy's documentation!
    spatial-raster
    spatial-vector
 
-
-
-Indices and tables
+Indices and Tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+About Earthpy's Documents
```

Comment:
hey @mcosmos12 i'm sorry i think you were lead a bit astray yesterday. What i suggested to @jlpalomino  that you do is create an introduction to the earthpy package following what rasterio has [here](https://rasterio.readthedocs.io/en/latest/)

Or geopandas, [here](https://geopandas.readthedocs.io/en/latest/)

The text below will end up BELOW the table of contents on the read the docs landing page and it doesn't actually help the user understand what earthpy is. So i can not accept this fix. I'm going to suggest you remove everything below here from this pr.